invenio_search_ui/assets/js/invenio_react_search_ui/App.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2020 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import { Container, Grid } from "semantic-ui-react";
+import {
+  ReactSearchKit,
+  SearchBar,
+  BucketAggregation,
+  EmptyResults,
+  Error,
+  ResultsLoader,
+  withState
+} from "react-searchkit";
+import { Results } from "./Results";
+import { InvenioSearchApi } from "react-searchkit";
+
+import "semantic-ui-css/semantic.min.css";
```

Comment:
note: this will have to be removed when integrated in the global Invenio theme

Reply:
yes, I'm waiting for the theme to be available to remove that